lib/ui/painting.dart

```diff
@@ -461,6 +486,42 @@ class Paint {
     _data.setInt32(_kStrokeCapOffset, encoded, _kFakeHostEndian);
   }
 
+  /// The kind of finish to place on the joins between segments.
+  ///
+  /// This applies to paths drawn when [style] is set to [PaintingStyle.stroke],
+  /// It does not apply to points drawn as lines with [Canvas.drawPoints].
+  ///
+  /// Defaults to [StrokeJoin.miter], i.e. sharp corners.  See also
+  /// [strokeMiterLimit] to control when miters are replaced by bevels.
+  StrokeJoin get strokeJoin {
+    return StrokeJoin.values[_data.getInt32(_kStrokeJoinOffset, _kFakeHostEndian)];
+  }
+  set strokeJoin(StrokeJoin value) {
+    assert(value != null);
+    final int encoded = value.index;
+    _data.setInt32(_kStrokeJoinOffset, encoded, _kFakeHostEndian);
+  }
+
+  /// The limit for miters to be drawn on segments when the join is set to
+  /// [StrokeJoin.miter] and the [style] is set to [PaintingStyle.stroke]. If
+  /// this limit is exceeded, then a [StrokeJoin.bevel] join will be drawn
+  /// instead.  Note that this may cause some 'popping' of the corners of a
```

Comment:
nit: single space after period, not double
avoid "Note that" (in this case you can probably replace ".  Note that" with just a semicolon).